apps/studio/components/interfaces/Settings/Logs/LogsQueryPanel.tsx

```diff
@@ -238,97 +238,93 @@ const LogsQueryPanel = ({
               </div>
             </div>
           </div>
-          <div>
-            <div className="flex items-center gap-4">
-              <div className="flex items-center gap-2">
-                <SidePanel
-                  size="large"
-                  header={
-                    <div className="flex flex-row justify-between items-center">
-                      <h3>Field Reference</h3>
-                      <Button
-                        type="text"
-                        className="px-1"
-                        onClick={() => setShowReference(false)}
-                        icon={<X size={18} strokeWidth={1.5} />}
-                      />
-                    </div>
-                  }
-                  visible={showReference}
-                  cancelText="Close"
-                  onCancel={() => setShowReference(false)}
-                  hideFooter
-                  triggerElement={
-                    <Button
-                      asChild // ?: we don't want a button inside a button
-                      type="default"
-                      onClick={() => setShowReference(true)}
-                      icon={<BookOpen />}
-                      className="px-2"
-                    >
-                      <span>Field Reference</span>
-                    </Button>
-                  }
+          {dataSource === 'logs' && (
+            <SidePanel
+              size="large"
+              header={
+                <div className="flex flex-row justify-between items-center">
+                  <h3>Field Reference</h3>
+                  <Button
+                    type="text"
+                    className="px-1"
+                    onClick={() => setShowReference(false)}
+                    icon={<X size={18} strokeWidth={1.5} />}
```

Comment:
can remove icon size and strokeWidth for button icon property - let the button style it

```suggestion
                    icon={<X />}
```